include/TrustWalletCore/TWCoinType.h

```diff
@@ -43,6 +43,7 @@ enum TWCoinType {
     TWCoinTypeDigiByte = 20,
     TWCoinTypeDogecoin = 3,
     TWCoinTypeEOS = 194,
+    TWCoinTypeWAX = 194001,
```

Comment:
Where is this coinID coming from? We usually take coinIDs only from https://github.com/satoshilabs/slips/blob/master/slip-0044.md, to make sure that we use standard ones, and don't use a value which is used by someone else. But currently WAX is not present there. We should ask them to create a PR for it.

Reply:
If this is just 'internal' ID, I suggest using 10000194 (we use such 8 digits for internally made up IDs).

Reply:
I just opened a pull request for slip-0044: https://github.com/satoshilabs/slips/pull/1473/files

Reply:
Ok, it was merged, so now this is a legit value :)